iron-ajax.html

```diff
@@ -126,7 +126,7 @@
        * and a `Content-Type` header is specified in the `headers` attribute,
        * the `headers` attribute value will take precedence.
        *
-       * @attribute contenttype
+       * @attribute content-type
```

Comment:
I guess this is just a misleading annotation. I saw `attribute` and thought I should use the attribute version. I will just use whatever is better.